README.md

```diff
@@ -131,6 +131,11 @@ By default the server starts with ML features enabled. To disable running ML fea
 - ML dependecies are installed by default. If you wish to not install them run `DISABLE_ML_FEATURES=true mix deps.get`
 - For example to start the phoenix server in dev mode without ML features run `DISABLE_ML_FEATURES=true iex -S mix phx.server`
 
+## LLama Version
```

Comment:
Please add a sentence or two about https://ollama.com too. Put yourself in the shoes of a potential new ANIMINA developer. What does he/she need to know to get the system up and running? Add links when it makes there lives easier.